frontend/debugger-implementation.js

```diff
@@ -721,7 +721,8 @@ function flattenSignalGraph(nodes)
 	function addAllToDict(node)
 	{
 		nodesById[node.id] = node;
-		node.kids.forEach(addAllToDict);
+		if (! node.isOutput)
```

Comment:
Two style nits (not sure if they're in line with the elm style):

1. put the `!` next to `node.isOutput` without a space, so `if (!node.isOutput)`
2. Add curly braces, even though it's a one liner. It sucks super hard when you need edit a conditional and forget to add curly braces that probably should have been there to begin with. I've had dumb bugs with this more than once and decided "never again"